bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI.Core/UserProfiles/IAccessManagementClient.cs

```diff
@@ -1,11 +1,14 @@
 ﻿using Altinn.Authentication.UI.Core.Common.Models;
+using Altinn.Authentication.UI.Core.SystemRegister;
 using Altinn.Platform.Register.Models;
 
 namespace Altinn.Authentication.UI.Core.UserProfiles;
 
-public interface IPartyClient
+public interface IAccessManagementClient
```

Comment:
Have we tested that this change does not impact the existing profile information sent to the frontend?

Reply:
It's just a namechange, since the client does more than lookup party info